todolist/src/api/index.js

```diff
@@ -0,0 +1,37 @@
+import axios from 'axios';
+
+const api = axios.create({
+  baseURL: 'http://localhost:8080',
+  timeout: 1000,
+  headers: { 'Content-Type': `application/json` },
+});
```

Comment:
```suggestion
const api = axios.create({
  baseURL: 'http://localhost:8080',
  timeout: 1000,
  headers: { 'Content-Type': 'application/json' },
});
```
사소하지만, 단순 문자열을 입력할 때에는 single quote로 사용해주세요.